diff --git a/dist/esm/index.js b/dist/esm/index.js
index 41467bb1f1d381233fb12204521e59763211689c..6252aecffd0ad877d7aff91c1a22134755f117bf 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -1362,7 +1362,7 @@ var useKeyPress = (keyCode = null, options = { actInsideInputWithModifier: true
                 const keyOrCode = useKeyOrCode(event.code, keysToWatch);
                 pressedKeys.current.add(event[keyOrCode]);
                 if (isMatchingKey(keyCodes, pressedKeys.current, false)) {
-                    event.preventDefault();
+                    // event.preventDefault();
                     setKeyPressed(true);
                 }
             };
diff --git a/dist/esm/index.mjs b/dist/esm/index.mjs
index 15a9d1617b31b7153611601c711152beb9ce4148..5e90b6f1f26ab8d247d5872bae630baec5e9e669 100644
--- a/dist/esm/index.mjs
+++ b/dist/esm/index.mjs
@@ -1362,7 +1362,7 @@ var useKeyPress = (keyCode = null, options = { actInsideInputWithModifier: true
                 const keyOrCode = useKeyOrCode(event.code, keysToWatch);
                 pressedKeys.current.add(event[keyOrCode]);
                 if (isMatchingKey(keyCodes, pressedKeys.current, false)) {
-                    event.preventDefault();
+                    // event.preventDefault();
                     setKeyPressed(true);
                 }
             };
