diff --git a/esm/vs/basic-languages/abap/abap.contribution.d.ts b/esm/vs/basic-languages/abap/abap.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/abap/abap.contribution.js b/esm/vs/basic-languages/abap/abap.contribution.js
deleted file mode 100644
index 9425008294f50b39e9e35f716164ec92166d6315..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/abap/abap.js b/esm/vs/basic-languages/abap/abap.js
deleted file mode 100644
index a2df71da72d6f2a24721206a1856abb516f79d20..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/apex/apex.contribution.d.ts b/esm/vs/basic-languages/apex/apex.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/apex/apex.contribution.js b/esm/vs/basic-languages/apex/apex.contribution.js
deleted file mode 100644
index 3a2a93b73361fb2d296f59d7d4f824d25d54febe..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/apex/apex.js b/esm/vs/basic-languages/apex/apex.js
deleted file mode 100644
index e0722f8bccbb618e75cfed3ec2f28a4db198b595..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/azcli/azcli.contribution.d.ts b/esm/vs/basic-languages/azcli/azcli.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/azcli/azcli.contribution.js b/esm/vs/basic-languages/azcli/azcli.contribution.js
deleted file mode 100644
index 899ee4aa65f028f5fdc5c9a7551e165111b3f8a6..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/azcli/azcli.js b/esm/vs/basic-languages/azcli/azcli.js
deleted file mode 100644
index 886cd51f2ff117cd7a45216ae91bb2b48d360d24..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/bat/bat.contribution.d.ts b/esm/vs/basic-languages/bat/bat.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/bat/bat.contribution.js b/esm/vs/basic-languages/bat/bat.contribution.js
deleted file mode 100644
index 7482624d4a5ada37bea863957b3afa27b9c459c0..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/bat/bat.js b/esm/vs/basic-languages/bat/bat.js
deleted file mode 100644
index e94090d0350d20f036d25617b6065a63a8a8f1b2..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/bicep/bicep.contribution.d.ts b/esm/vs/basic-languages/bicep/bicep.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/bicep/bicep.contribution.js b/esm/vs/basic-languages/bicep/bicep.contribution.js
deleted file mode 100644
index 13bf6c15331c31cfe7f8134485774291fadab58b..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/bicep/bicep.js b/esm/vs/basic-languages/bicep/bicep.js
deleted file mode 100644
index 11219c422e41df2c27ce255ae68101d6a132e67a..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cameligo/cameligo.contribution.d.ts b/esm/vs/basic-languages/cameligo/cameligo.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cameligo/cameligo.contribution.js b/esm/vs/basic-languages/cameligo/cameligo.contribution.js
deleted file mode 100644
index 218d12ad5c379d93cd6aeb31bdf6f18d57463d04..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cameligo/cameligo.js b/esm/vs/basic-languages/cameligo/cameligo.js
deleted file mode 100644
index 7a07aa052678454e66ad202ba60225a9f7bafe05..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/clojure/clojure.contribution.d.ts b/esm/vs/basic-languages/clojure/clojure.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/clojure/clojure.contribution.js b/esm/vs/basic-languages/clojure/clojure.contribution.js
deleted file mode 100644
index 01b8eac4a513f38ef6760f1ba8568d62882dcc84..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/clojure/clojure.js b/esm/vs/basic-languages/clojure/clojure.js
deleted file mode 100644
index abcbffe391018d75afb5ae34df7dd886d716aec4..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/coffee/coffee.contribution.d.ts b/esm/vs/basic-languages/coffee/coffee.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/coffee/coffee.contribution.js b/esm/vs/basic-languages/coffee/coffee.contribution.js
deleted file mode 100644
index f39b3fac251f82df090c555d794c447714bf754c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/coffee/coffee.js b/esm/vs/basic-languages/coffee/coffee.js
deleted file mode 100644
index 04efec99272d08523a8a32e4f5876b8af13107da..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cpp/cpp.contribution.d.ts b/esm/vs/basic-languages/cpp/cpp.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cpp/cpp.contribution.js b/esm/vs/basic-languages/cpp/cpp.contribution.js
deleted file mode 100644
index bebe38ad9b971621924a87c0ec48207a6e743b81..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cpp/cpp.js b/esm/vs/basic-languages/cpp/cpp.js
deleted file mode 100644
index 8aa3dd6baf31cd36b6a50f93ae4c4c57e9c5a492..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/csharp/csharp.contribution.d.ts b/esm/vs/basic-languages/csharp/csharp.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/csharp/csharp.contribution.js b/esm/vs/basic-languages/csharp/csharp.contribution.js
deleted file mode 100644
index 976451bc504e1b6eed9e4195b6c2769c626b378a..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/csharp/csharp.js b/esm/vs/basic-languages/csharp/csharp.js
deleted file mode 100644
index e8431872f96cb071f1d74a56ab82a9fc895b386f..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/csp/csp.contribution.d.ts b/esm/vs/basic-languages/csp/csp.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/csp/csp.contribution.js b/esm/vs/basic-languages/csp/csp.contribution.js
deleted file mode 100644
index 9f6a88bcd8a127ced234adec30c43c374369cb82..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/csp/csp.js b/esm/vs/basic-languages/csp/csp.js
deleted file mode 100644
index 647a634a153c80a8e6ca1899a10a2dfd8da28e29..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/css/css.contribution.d.ts b/esm/vs/basic-languages/css/css.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/css/css.contribution.js b/esm/vs/basic-languages/css/css.contribution.js
deleted file mode 100644
index 07916e02cd519f21efd7e23775c114e5d12cf444..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/css/css.js b/esm/vs/basic-languages/css/css.js
deleted file mode 100644
index 3c036178d3d701ebdf5576e4c96620e1c43c8b7a..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cypher/cypher.contribution.d.ts b/esm/vs/basic-languages/cypher/cypher.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cypher/cypher.contribution.js b/esm/vs/basic-languages/cypher/cypher.contribution.js
deleted file mode 100644
index a7a4a21efac9475850b7a650e7326bbe5282cf22..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/cypher/cypher.js b/esm/vs/basic-languages/cypher/cypher.js
deleted file mode 100644
index cfe07672aca89d919810251c3abf5cc4caa9c7e4..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/dart/dart.contribution.d.ts b/esm/vs/basic-languages/dart/dart.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/dart/dart.contribution.js b/esm/vs/basic-languages/dart/dart.contribution.js
deleted file mode 100644
index 97367795ac93404857ddebfe8c173693feb6a8f5..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/dart/dart.js b/esm/vs/basic-languages/dart/dart.js
deleted file mode 100644
index 8a00804ae8cdb18918347949313d5e213b354479..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/dockerfile/dockerfile.contribution.d.ts b/esm/vs/basic-languages/dockerfile/dockerfile.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/dockerfile/dockerfile.contribution.js b/esm/vs/basic-languages/dockerfile/dockerfile.contribution.js
deleted file mode 100644
index 5294152d47c7ea4805d917e89a6a7d4fc6d2c837..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/dockerfile/dockerfile.js b/esm/vs/basic-languages/dockerfile/dockerfile.js
deleted file mode 100644
index 376c9d8817bf2242909c886f065aeefaa61e5b90..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ecl/ecl.contribution.d.ts b/esm/vs/basic-languages/ecl/ecl.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ecl/ecl.contribution.js b/esm/vs/basic-languages/ecl/ecl.contribution.js
deleted file mode 100644
index 3dec7b9c1e0c4298566e8aa7097417c39a000245..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ecl/ecl.js b/esm/vs/basic-languages/ecl/ecl.js
deleted file mode 100644
index 40a0e55dcddf8817edb873399cb82798a515aeaa..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/elixir/elixir.contribution.d.ts b/esm/vs/basic-languages/elixir/elixir.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/elixir/elixir.contribution.js b/esm/vs/basic-languages/elixir/elixir.contribution.js
deleted file mode 100644
index 2cfa678a8d3fee6d40029abbafad835bfe65eb6d..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/elixir/elixir.js b/esm/vs/basic-languages/elixir/elixir.js
deleted file mode 100644
index 2e782ebe8fedc157aa5178499e1000a05f334f22..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/flow9/flow9.contribution.d.ts b/esm/vs/basic-languages/flow9/flow9.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/flow9/flow9.contribution.js b/esm/vs/basic-languages/flow9/flow9.contribution.js
deleted file mode 100644
index 0e66850e2b2dc5991ca32b35c871a0da49d716fb..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/flow9/flow9.js b/esm/vs/basic-languages/flow9/flow9.js
deleted file mode 100644
index d620cc5383e410ec82a1c81b6b6a2c3385016f41..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/freemarker2/freemarker2.contribution.d.ts b/esm/vs/basic-languages/freemarker2/freemarker2.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/freemarker2/freemarker2.contribution.js b/esm/vs/basic-languages/freemarker2/freemarker2.contribution.js
deleted file mode 100644
index 4fdf000370955761ccecc8f0d94c96d853887451..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/freemarker2/freemarker2.js b/esm/vs/basic-languages/freemarker2/freemarker2.js
deleted file mode 100644
index 98409923310f6f761bba82d0b522d4df29f40cdb..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/fsharp/fsharp.contribution.d.ts b/esm/vs/basic-languages/fsharp/fsharp.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/fsharp/fsharp.contribution.js b/esm/vs/basic-languages/fsharp/fsharp.contribution.js
deleted file mode 100644
index ed6ac1ca980581978e476505bac4bb4142599d60..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/fsharp/fsharp.js b/esm/vs/basic-languages/fsharp/fsharp.js
deleted file mode 100644
index a507e5985279eae940c065eaab0b45b518cc4c72..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/go/go.contribution.d.ts b/esm/vs/basic-languages/go/go.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/go/go.contribution.js b/esm/vs/basic-languages/go/go.contribution.js
deleted file mode 100644
index 9e9a8a930321d077c845e43c70d1a28f691434f4..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/go/go.js b/esm/vs/basic-languages/go/go.js
deleted file mode 100644
index e19ad5d20144fc5f3f886c7bab14e4e87d480dd3..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/graphql/graphql.contribution.d.ts b/esm/vs/basic-languages/graphql/graphql.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/graphql/graphql.contribution.js b/esm/vs/basic-languages/graphql/graphql.contribution.js
deleted file mode 100644
index cef4a7d0c1bcf4694d186805cbf8be34cc532629..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/graphql/graphql.js b/esm/vs/basic-languages/graphql/graphql.js
deleted file mode 100644
index 23cabb12993994b1859ea435af56066c6af514b7..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/handlebars/handlebars.contribution.d.ts b/esm/vs/basic-languages/handlebars/handlebars.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/handlebars/handlebars.contribution.js b/esm/vs/basic-languages/handlebars/handlebars.contribution.js
deleted file mode 100644
index 6710ec6f5c1d404d9175cf4e22522f21c2e82519..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/handlebars/handlebars.js b/esm/vs/basic-languages/handlebars/handlebars.js
deleted file mode 100644
index cf918f44b6cbf906d2c14440e43077dd691317ef..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/hcl/hcl.contribution.d.ts b/esm/vs/basic-languages/hcl/hcl.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/hcl/hcl.contribution.js b/esm/vs/basic-languages/hcl/hcl.contribution.js
deleted file mode 100644
index eac3bb7387fb867d06867c6cdf705d45b46747ca..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/hcl/hcl.js b/esm/vs/basic-languages/hcl/hcl.js
deleted file mode 100644
index 116f21b6355c0c7c9197f9a00201ee204c55db4e..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/html/html.contribution.d.ts b/esm/vs/basic-languages/html/html.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/html/html.contribution.js b/esm/vs/basic-languages/html/html.contribution.js
deleted file mode 100644
index 87f1d0974b7d647d3704cb99e1a17b12d19d5564..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/html/html.js b/esm/vs/basic-languages/html/html.js
deleted file mode 100644
index 3d8bda15a0286562c595bc59ac34437065696866..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ini/ini.contribution.d.ts b/esm/vs/basic-languages/ini/ini.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ini/ini.contribution.js b/esm/vs/basic-languages/ini/ini.contribution.js
deleted file mode 100644
index a431d7742b85f1ac1098631000a4b735429107e3..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ini/ini.js b/esm/vs/basic-languages/ini/ini.js
deleted file mode 100644
index 81f1429a66a75712c0bf0669c45e2075a7859548..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/java/java.contribution.d.ts b/esm/vs/basic-languages/java/java.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/java/java.contribution.js b/esm/vs/basic-languages/java/java.contribution.js
deleted file mode 100644
index c3294d26a13c0f536c1a64e74a279eaabd0e2d2d..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/java/java.js b/esm/vs/basic-languages/java/java.js
deleted file mode 100644
index 28f7c9529e297ec338d1ec5d0f9cdc52a42b726d..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/javascript/javascript.contribution.d.ts b/esm/vs/basic-languages/javascript/javascript.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/javascript/javascript.contribution.js b/esm/vs/basic-languages/javascript/javascript.contribution.js
deleted file mode 100644
index 941a31f86334ea5c3db1c14e3fc51e408f573cff..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/javascript/javascript.js b/esm/vs/basic-languages/javascript/javascript.js
deleted file mode 100644
index 69c2c92a45ef4289e2f9ac2215975fb43150165e..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/julia/julia.contribution.d.ts b/esm/vs/basic-languages/julia/julia.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/julia/julia.contribution.js b/esm/vs/basic-languages/julia/julia.contribution.js
deleted file mode 100644
index 97acb2fc758a65cff3cf11eeafadb0cd1e900184..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/julia/julia.js b/esm/vs/basic-languages/julia/julia.js
deleted file mode 100644
index bf6ee400f4469b85794c79d915f6571de5309ac3..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/kotlin/kotlin.contribution.d.ts b/esm/vs/basic-languages/kotlin/kotlin.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/kotlin/kotlin.contribution.js b/esm/vs/basic-languages/kotlin/kotlin.contribution.js
deleted file mode 100644
index d90ae4199afa6c6fbc3f2f396b073959afab0ea5..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/kotlin/kotlin.js b/esm/vs/basic-languages/kotlin/kotlin.js
deleted file mode 100644
index ba015b226e20fd85f442c3bb40dcc2a3a3decd53..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/less/less.contribution.d.ts b/esm/vs/basic-languages/less/less.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/less/less.contribution.js b/esm/vs/basic-languages/less/less.contribution.js
deleted file mode 100644
index 932183e7b07503f51b4889fe8c5322b2ce4d1d5f..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/less/less.js b/esm/vs/basic-languages/less/less.js
deleted file mode 100644
index 839024ad07e8dd5a8236c43e50c1c8bc3c09e18c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/lexon/lexon.contribution.d.ts b/esm/vs/basic-languages/lexon/lexon.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/lexon/lexon.contribution.js b/esm/vs/basic-languages/lexon/lexon.contribution.js
deleted file mode 100644
index aa99753cf257626dddb05bdd121bde5b36d4c391..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/lexon/lexon.js b/esm/vs/basic-languages/lexon/lexon.js
deleted file mode 100644
index ab0c919c19fcd54dbb1bee6af1b69fad47afd6b7..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/liquid/liquid.contribution.d.ts b/esm/vs/basic-languages/liquid/liquid.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/liquid/liquid.contribution.js b/esm/vs/basic-languages/liquid/liquid.contribution.js
deleted file mode 100644
index 7a2b13c397b61f2e6752061dd1f1cb885931a0a9..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/liquid/liquid.js b/esm/vs/basic-languages/liquid/liquid.js
deleted file mode 100644
index 1dcd73a7128a568b04a985cafb2226a68855d7b7..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/lua/lua.contribution.d.ts b/esm/vs/basic-languages/lua/lua.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/lua/lua.contribution.js b/esm/vs/basic-languages/lua/lua.contribution.js
deleted file mode 100644
index 4740595ccf11375d5c1c2e4d8d9f82f781a832f6..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/lua/lua.js b/esm/vs/basic-languages/lua/lua.js
deleted file mode 100644
index 1bf41cb00427027c252746780a97977d21e764d5..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/m3/m3.contribution.d.ts b/esm/vs/basic-languages/m3/m3.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/m3/m3.contribution.js b/esm/vs/basic-languages/m3/m3.contribution.js
deleted file mode 100644
index b786b3ed60be2f46b2c0d76e4171b740ce591fce..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/m3/m3.js b/esm/vs/basic-languages/m3/m3.js
deleted file mode 100644
index eadd7e2694a60da6738757410c478ee3e1c53f16..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/markdown/markdown.contribution.d.ts b/esm/vs/basic-languages/markdown/markdown.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/markdown/markdown.contribution.js b/esm/vs/basic-languages/markdown/markdown.contribution.js
deleted file mode 100644
index 699fc4522b951969aed40cfd3a28d3d94278ba4e..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/markdown/markdown.js b/esm/vs/basic-languages/markdown/markdown.js
deleted file mode 100644
index 0250d2c9bed4408a206ec6d510f67fd152902fee..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mdx/mdx.contribution.d.ts b/esm/vs/basic-languages/mdx/mdx.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mdx/mdx.contribution.js b/esm/vs/basic-languages/mdx/mdx.contribution.js
deleted file mode 100644
index 533224217138679ccfc5bc7f4220847a341e5734..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mdx/mdx.js b/esm/vs/basic-languages/mdx/mdx.js
deleted file mode 100644
index c8b5f42e7f590e40249c56767d6a90c9b682767c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mips/mips.contribution.d.ts b/esm/vs/basic-languages/mips/mips.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mips/mips.contribution.js b/esm/vs/basic-languages/mips/mips.contribution.js
deleted file mode 100644
index b52c3d295521de467ffab33b5b5497e4209c58fc..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mips/mips.js b/esm/vs/basic-languages/mips/mips.js
deleted file mode 100644
index acb89311356a67a4a9407f815f09560a1e4cf5e1..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/monaco.contribution.js b/esm/vs/basic-languages/monaco.contribution.js
index f67e19dda72c587c1e07990e2e833e6424d51e0c..a9cba84cea4e914956f80bca8a43f3a81f85b820 100644
--- a/esm/vs/basic-languages/monaco.contribution.js
+++ b/esm/vs/basic-languages/monaco.contribution.js
@@ -7,83 +7,3 @@ import '../editor/editor.api.js';
  *-----------------------------------------------------------------------------*/
 
 // src/basic-languages/monaco.contribution.ts
-import "./abap/abap.contribution.js";
-import "./apex/apex.contribution.js";
-import "./azcli/azcli.contribution.js";
-import "./bat/bat.contribution.js";
-import "./bicep/bicep.contribution.js";
-import "./cameligo/cameligo.contribution.js";
-import "./clojure/clojure.contribution.js";
-import "./coffee/coffee.contribution.js";
-import "./cpp/cpp.contribution.js";
-import "./csharp/csharp.contribution.js";
-import "./csp/csp.contribution.js";
-import "./css/css.contribution.js";
-import "./cypher/cypher.contribution.js";
-import "./dart/dart.contribution.js";
-import "./dockerfile/dockerfile.contribution.js";
-import "./ecl/ecl.contribution.js";
-import "./elixir/elixir.contribution.js";
-import "./flow9/flow9.contribution.js";
-import "./fsharp/fsharp.contribution.js";
-import "./freemarker2/freemarker2.contribution.js";
-import "./go/go.contribution.js";
-import "./graphql/graphql.contribution.js";
-import "./handlebars/handlebars.contribution.js";
-import "./hcl/hcl.contribution.js";
-import "./html/html.contribution.js";
-import "./ini/ini.contribution.js";
-import "./java/java.contribution.js";
-import "./javascript/javascript.contribution.js";
-import "./julia/julia.contribution.js";
-import "./kotlin/kotlin.contribution.js";
-import "./less/less.contribution.js";
-import "./lexon/lexon.contribution.js";
-import "./lua/lua.contribution.js";
-import "./liquid/liquid.contribution.js";
-import "./m3/m3.contribution.js";
-import "./markdown/markdown.contribution.js";
-import "./mdx/mdx.contribution.js";
-import "./mips/mips.contribution.js";
-import "./msdax/msdax.contribution.js";
-import "./mysql/mysql.contribution.js";
-import "./objective-c/objective-c.contribution.js";
-import "./pascal/pascal.contribution.js";
-import "./pascaligo/pascaligo.contribution.js";
-import "./perl/perl.contribution.js";
-import "./pgsql/pgsql.contribution.js";
-import "./php/php.contribution.js";
-import "./pla/pla.contribution.js";
-import "./postiats/postiats.contribution.js";
-import "./powerquery/powerquery.contribution.js";
-import "./powershell/powershell.contribution.js";
-import "./protobuf/protobuf.contribution.js";
-import "./pug/pug.contribution.js";
-import "./python/python.contribution.js";
-import "./qsharp/qsharp.contribution.js";
-import "./r/r.contribution.js";
-import "./razor/razor.contribution.js";
-import "./redis/redis.contribution.js";
-import "./redshift/redshift.contribution.js";
-import "./restructuredtext/restructuredtext.contribution.js";
-import "./ruby/ruby.contribution.js";
-import "./rust/rust.contribution.js";
-import "./sb/sb.contribution.js";
-import "./scala/scala.contribution.js";
-import "./scheme/scheme.contribution.js";
-import "./scss/scss.contribution.js";
-import "./shell/shell.contribution.js";
-import "./solidity/solidity.contribution.js";
-import "./sophia/sophia.contribution.js";
-import "./sparql/sparql.contribution.js";
-import "./sql/sql.contribution.js";
-import "./st/st.contribution.js";
-import "./swift/swift.contribution.js";
-import "./systemverilog/systemverilog.contribution.js";
-import "./tcl/tcl.contribution.js";
-import "./twig/twig.contribution.js";
-import "./typescript/typescript.contribution.js";
-import "./vb/vb.contribution.js";
-import "./wgsl/wgsl.contribution.js";
-import "./xml/xml.contribution.js";
-import "./yaml/yaml.contribution.js";
diff --git a/esm/vs/basic-languages/msdax/msdax.contribution.d.ts b/esm/vs/basic-languages/msdax/msdax.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/msdax/msdax.contribution.js b/esm/vs/basic-languages/msdax/msdax.contribution.js
deleted file mode 100644
index 98eaf0c031a6ec7e6d8ce7e843cdeb966b309245..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/msdax/msdax.js b/esm/vs/basic-languages/msdax/msdax.js
deleted file mode 100644
index 25fff767ff5b479c7ee97c40463f18e8958bc838..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mysql/mysql.contribution.d.ts b/esm/vs/basic-languages/mysql/mysql.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mysql/mysql.contribution.js b/esm/vs/basic-languages/mysql/mysql.contribution.js
deleted file mode 100644
index f1f23511672a3517501203d6f005e19cc563a94a..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/mysql/mysql.js b/esm/vs/basic-languages/mysql/mysql.js
deleted file mode 100644
index ca148fe9d08c397578d5561b1fb42aec5dce79dc..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/objective-c/objective-c.contribution.d.ts b/esm/vs/basic-languages/objective-c/objective-c.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/objective-c/objective-c.contribution.js b/esm/vs/basic-languages/objective-c/objective-c.contribution.js
deleted file mode 100644
index e2c24474a8b3815763407ddcd519a71e16f35954..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/objective-c/objective-c.js b/esm/vs/basic-languages/objective-c/objective-c.js
deleted file mode 100644
index d9a8bffaa34f59432fff3e900928833ffef1d279..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pascal/pascal.contribution.d.ts b/esm/vs/basic-languages/pascal/pascal.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pascal/pascal.contribution.js b/esm/vs/basic-languages/pascal/pascal.contribution.js
deleted file mode 100644
index b8fce3307edc14e65e767530c5645b6cfea9de6c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pascal/pascal.js b/esm/vs/basic-languages/pascal/pascal.js
deleted file mode 100644
index 5450acfcd8059ffc5f9f593b631e21a6bc854bd3..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pascaligo/pascaligo.contribution.d.ts b/esm/vs/basic-languages/pascaligo/pascaligo.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pascaligo/pascaligo.contribution.js b/esm/vs/basic-languages/pascaligo/pascaligo.contribution.js
deleted file mode 100644
index bfe1cbf2745702b02b0d0508d3e00f93685b73d0..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pascaligo/pascaligo.js b/esm/vs/basic-languages/pascaligo/pascaligo.js
deleted file mode 100644
index 027f53feb928ed1e9bcde8f0f1f0eacbf4cc5e5d..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/perl/perl.contribution.d.ts b/esm/vs/basic-languages/perl/perl.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/perl/perl.contribution.js b/esm/vs/basic-languages/perl/perl.contribution.js
deleted file mode 100644
index f32ed409a1e844c6a5b286500469ce6dcc0c1d5f..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/perl/perl.js b/esm/vs/basic-languages/perl/perl.js
deleted file mode 100644
index c5334e0c567b0517c73619d1640aff37211ddb97..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pgsql/pgsql.contribution.d.ts b/esm/vs/basic-languages/pgsql/pgsql.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pgsql/pgsql.contribution.js b/esm/vs/basic-languages/pgsql/pgsql.contribution.js
deleted file mode 100644
index 6932b80611fdc85366cf71169acffd8446cb82fe..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pgsql/pgsql.js b/esm/vs/basic-languages/pgsql/pgsql.js
deleted file mode 100644
index f77de1da12979d3c67a523763f03d3e3e2cd7d51..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/php/php.contribution.d.ts b/esm/vs/basic-languages/php/php.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/php/php.contribution.js b/esm/vs/basic-languages/php/php.contribution.js
deleted file mode 100644
index 2df16a1df76491c512b1c0bb88f1fbfb7dbc7219..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/php/php.js b/esm/vs/basic-languages/php/php.js
deleted file mode 100644
index 443dd4f3a32a0db68255c4e0f89ee9abded742dc..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pla/pla.contribution.d.ts b/esm/vs/basic-languages/pla/pla.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pla/pla.contribution.js b/esm/vs/basic-languages/pla/pla.contribution.js
deleted file mode 100644
index 8fbd2487f3f0e3224773bd7a3c6f2c79e3c944b8..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pla/pla.js b/esm/vs/basic-languages/pla/pla.js
deleted file mode 100644
index 6b5f8c9bf7fe4155bad4ba8f724b220d2b32d6f1..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/postiats/postiats.contribution.d.ts b/esm/vs/basic-languages/postiats/postiats.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/postiats/postiats.contribution.js b/esm/vs/basic-languages/postiats/postiats.contribution.js
deleted file mode 100644
index 06be2c9218738be85a908c46c00fc6341d5cac0c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/postiats/postiats.js b/esm/vs/basic-languages/postiats/postiats.js
deleted file mode 100644
index 74493e2ee05a92e164e6d76f81028f401681efe4..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/powerquery/powerquery.contribution.d.ts b/esm/vs/basic-languages/powerquery/powerquery.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/powerquery/powerquery.contribution.js b/esm/vs/basic-languages/powerquery/powerquery.contribution.js
deleted file mode 100644
index 8eadfd896893a7fd9ab4d50cfc4c2de2ab4a2bfb..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/powerquery/powerquery.js b/esm/vs/basic-languages/powerquery/powerquery.js
deleted file mode 100644
index f1004cedf3ccdb5e9f7ced9d6d8e35ee22f50ee0..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/powershell/powershell.contribution.d.ts b/esm/vs/basic-languages/powershell/powershell.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/powershell/powershell.contribution.js b/esm/vs/basic-languages/powershell/powershell.contribution.js
deleted file mode 100644
index 2fb508e6415d697d36a013c2fafeb0d87144bc09..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/powershell/powershell.js b/esm/vs/basic-languages/powershell/powershell.js
deleted file mode 100644
index 3d5678a2353d5ddab73eeaa6b03ef0fd9428f859..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/protobuf/protobuf.contribution.d.ts b/esm/vs/basic-languages/protobuf/protobuf.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/protobuf/protobuf.contribution.js b/esm/vs/basic-languages/protobuf/protobuf.contribution.js
deleted file mode 100644
index 0e49ae8e4a300b57a4af2f1beb15a3e9fc8f0b31..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/protobuf/protobuf.js b/esm/vs/basic-languages/protobuf/protobuf.js
deleted file mode 100644
index eb6ce68f21c02a344f1a7fc24f119c520738022f..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pug/pug.contribution.d.ts b/esm/vs/basic-languages/pug/pug.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pug/pug.contribution.js b/esm/vs/basic-languages/pug/pug.contribution.js
deleted file mode 100644
index 95e5144c3a3ec92e1310950db0223a2bc6c3e747..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/pug/pug.js b/esm/vs/basic-languages/pug/pug.js
deleted file mode 100644
index eab860f8caa617d0a209e9de60d2ada2e6776559..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/python/python.contribution.d.ts b/esm/vs/basic-languages/python/python.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/python/python.contribution.js b/esm/vs/basic-languages/python/python.contribution.js
deleted file mode 100644
index da037f8ab0d230541e42cb7fc35ae4d59d577dbf..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/python/python.js b/esm/vs/basic-languages/python/python.js
deleted file mode 100644
index 4613ea934e56fb6039ecf5ace815af3c02c2f460..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/qsharp/qsharp.contribution.d.ts b/esm/vs/basic-languages/qsharp/qsharp.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/qsharp/qsharp.contribution.js b/esm/vs/basic-languages/qsharp/qsharp.contribution.js
deleted file mode 100644
index 3825fc5f417950589ed0e2e92084237967fa0677..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/qsharp/qsharp.js b/esm/vs/basic-languages/qsharp/qsharp.js
deleted file mode 100644
index 2bf6009578c640754ac80ad3bd4b56b6dfef150e..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/r/r.contribution.d.ts b/esm/vs/basic-languages/r/r.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/r/r.contribution.js b/esm/vs/basic-languages/r/r.contribution.js
deleted file mode 100644
index 7530d3d28e9da8768c0863669930c91ead6254da..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/r/r.js b/esm/vs/basic-languages/r/r.js
deleted file mode 100644
index d8bab06949cecd12d5331aef49bb7a5af622b62c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/razor/razor.contribution.d.ts b/esm/vs/basic-languages/razor/razor.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/razor/razor.contribution.js b/esm/vs/basic-languages/razor/razor.contribution.js
deleted file mode 100644
index de56944ae9f570578d9073e9cb9f2d6be40d095a..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/razor/razor.js b/esm/vs/basic-languages/razor/razor.js
deleted file mode 100644
index 65d3fbd40023ab2b9e53f32744a3edf1f9dbb203..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/redis/redis.contribution.d.ts b/esm/vs/basic-languages/redis/redis.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/redis/redis.contribution.js b/esm/vs/basic-languages/redis/redis.contribution.js
deleted file mode 100644
index f7c3617efbe174a6bd53971dae60009826732386..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/redis/redis.js b/esm/vs/basic-languages/redis/redis.js
deleted file mode 100644
index 26981fa83b72dc37b1d74d40e75c7d8101544d63..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/redshift/redshift.contribution.d.ts b/esm/vs/basic-languages/redshift/redshift.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/redshift/redshift.contribution.js b/esm/vs/basic-languages/redshift/redshift.contribution.js
deleted file mode 100644
index 4ed448c4cf3bf1208582992a1f7a9aa827d5a038..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/redshift/redshift.js b/esm/vs/basic-languages/redshift/redshift.js
deleted file mode 100644
index f5112db9bd09efff86f6744db9827eec1dc80b9c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/restructuredtext/restructuredtext.contribution.d.ts b/esm/vs/basic-languages/restructuredtext/restructuredtext.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/restructuredtext/restructuredtext.contribution.js b/esm/vs/basic-languages/restructuredtext/restructuredtext.contribution.js
deleted file mode 100644
index 3735910d7e018d19abbd2ebb35badd58230d81a9..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/restructuredtext/restructuredtext.js b/esm/vs/basic-languages/restructuredtext/restructuredtext.js
deleted file mode 100644
index a5ffa447f72ce875995db0c1cf74888133493270..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ruby/ruby.contribution.d.ts b/esm/vs/basic-languages/ruby/ruby.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ruby/ruby.contribution.js b/esm/vs/basic-languages/ruby/ruby.contribution.js
deleted file mode 100644
index 75ca6a5d5cfb26f606a0033099855491631f8dd0..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/ruby/ruby.js b/esm/vs/basic-languages/ruby/ruby.js
deleted file mode 100644
index 828aff29e151186909569138a4d99f2633356d13..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/rust/rust.contribution.d.ts b/esm/vs/basic-languages/rust/rust.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/rust/rust.contribution.js b/esm/vs/basic-languages/rust/rust.contribution.js
deleted file mode 100644
index c620b1cbd3c6c78b8f2b9ad3fd48d5e53adfe721..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/rust/rust.js b/esm/vs/basic-languages/rust/rust.js
deleted file mode 100644
index abe213a534398ed909c4760b71ac4df8b92d7c02..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sb/sb.contribution.d.ts b/esm/vs/basic-languages/sb/sb.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sb/sb.contribution.js b/esm/vs/basic-languages/sb/sb.contribution.js
deleted file mode 100644
index 8f7896d54f0855711b58b74ca4e6faa9575b17f5..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sb/sb.js b/esm/vs/basic-languages/sb/sb.js
deleted file mode 100644
index 3ed3df5c6d4b4c02b6d714f3c744ba549ccaa5b0..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scala/scala.contribution.d.ts b/esm/vs/basic-languages/scala/scala.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scala/scala.contribution.js b/esm/vs/basic-languages/scala/scala.contribution.js
deleted file mode 100644
index 1b0fea29bee88bb2f3179a397f831ced097d478c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scala/scala.js b/esm/vs/basic-languages/scala/scala.js
deleted file mode 100644
index 94cd60baf19db5b0f9a9221db748a9844bf10f67..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scheme/scheme.contribution.d.ts b/esm/vs/basic-languages/scheme/scheme.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scheme/scheme.contribution.js b/esm/vs/basic-languages/scheme/scheme.contribution.js
deleted file mode 100644
index c74347a337cb2f4c0442b573140a081accb0fd01..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scheme/scheme.js b/esm/vs/basic-languages/scheme/scheme.js
deleted file mode 100644
index 259ad1ed2e13d536b753618068b7c6eaed2680cb..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scss/scss.contribution.d.ts b/esm/vs/basic-languages/scss/scss.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scss/scss.contribution.js b/esm/vs/basic-languages/scss/scss.contribution.js
deleted file mode 100644
index b9aeda6d543b5c98a707042068fc44901960da50..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/scss/scss.js b/esm/vs/basic-languages/scss/scss.js
deleted file mode 100644
index 539cb901618b2875c2d0e63d796e24e9e743166f..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/shell/shell.contribution.d.ts b/esm/vs/basic-languages/shell/shell.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/shell/shell.contribution.js b/esm/vs/basic-languages/shell/shell.contribution.js
deleted file mode 100644
index 26cffb64902fd124e7959fa9e798aef5ca0013d9..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/shell/shell.js b/esm/vs/basic-languages/shell/shell.js
deleted file mode 100644
index 22ec7569145c5bee0b2709a8db61ac30046a0f19..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/solidity/solidity.contribution.d.ts b/esm/vs/basic-languages/solidity/solidity.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/solidity/solidity.contribution.js b/esm/vs/basic-languages/solidity/solidity.contribution.js
deleted file mode 100644
index 0cd00be789f4f2fa087d280fca6e86a6ec6899f3..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/solidity/solidity.js b/esm/vs/basic-languages/solidity/solidity.js
deleted file mode 100644
index 01d88b4cda0d624a4503f0c5d6979c76fd1d8356..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sophia/sophia.contribution.d.ts b/esm/vs/basic-languages/sophia/sophia.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sophia/sophia.contribution.js b/esm/vs/basic-languages/sophia/sophia.contribution.js
deleted file mode 100644
index a7822145867f5cdb22053be0394fdcc4f89165aa..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sophia/sophia.js b/esm/vs/basic-languages/sophia/sophia.js
deleted file mode 100644
index 9a49566b7151f2d9ea9d5d0653a515b57e87fbb8..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sparql/sparql.contribution.d.ts b/esm/vs/basic-languages/sparql/sparql.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sparql/sparql.contribution.js b/esm/vs/basic-languages/sparql/sparql.contribution.js
deleted file mode 100644
index 72734595377706c585c009cb969d460072531694..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sparql/sparql.js b/esm/vs/basic-languages/sparql/sparql.js
deleted file mode 100644
index 403d73cc25c7fba7cf100e241d06652b02cd7a13..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sql/sql.contribution.d.ts b/esm/vs/basic-languages/sql/sql.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sql/sql.contribution.js b/esm/vs/basic-languages/sql/sql.contribution.js
deleted file mode 100644
index 99d221c211e428ebde5e148663f3bac3a1201db1..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/sql/sql.js b/esm/vs/basic-languages/sql/sql.js
deleted file mode 100644
index f940b70a22455d66e9d1a601a2404f33ffe5a964..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/st/st.contribution.d.ts b/esm/vs/basic-languages/st/st.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/st/st.contribution.js b/esm/vs/basic-languages/st/st.contribution.js
deleted file mode 100644
index 3727285b169f0b537fa60677d1bc6cf0531cef47..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/st/st.js b/esm/vs/basic-languages/st/st.js
deleted file mode 100644
index bed4ee6eb8ba6d9bc61fda4724fff1db99160202..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/swift/swift.contribution.d.ts b/esm/vs/basic-languages/swift/swift.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/swift/swift.contribution.js b/esm/vs/basic-languages/swift/swift.contribution.js
deleted file mode 100644
index feabe82a05cf13fae9e0d095b367e132b10a280c..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/swift/swift.js b/esm/vs/basic-languages/swift/swift.js
deleted file mode 100644
index 8a4eb5d7e011f3a1bc9393a501f4e44908b3105b..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/systemverilog/systemverilog.contribution.d.ts b/esm/vs/basic-languages/systemverilog/systemverilog.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/systemverilog/systemverilog.contribution.js b/esm/vs/basic-languages/systemverilog/systemverilog.contribution.js
deleted file mode 100644
index 68390ea71eaa55ecdf2f77ba2cca6d835cc90811..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/systemverilog/systemverilog.js b/esm/vs/basic-languages/systemverilog/systemverilog.js
deleted file mode 100644
index baa562147a39b8f65e213ad9fec236425ebd7748..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/tcl/tcl.contribution.d.ts b/esm/vs/basic-languages/tcl/tcl.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/tcl/tcl.contribution.js b/esm/vs/basic-languages/tcl/tcl.contribution.js
deleted file mode 100644
index ad11fe070e8323a42808422a272ac443ab4bd432..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/tcl/tcl.js b/esm/vs/basic-languages/tcl/tcl.js
deleted file mode 100644
index a8251627a92ba129f869552fa87448698652d5bb..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/twig/twig.contribution.d.ts b/esm/vs/basic-languages/twig/twig.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/twig/twig.contribution.js b/esm/vs/basic-languages/twig/twig.contribution.js
deleted file mode 100644
index 5c76887968d2aff02394675b8d6fd21788637d52..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/twig/twig.js b/esm/vs/basic-languages/twig/twig.js
deleted file mode 100644
index 336c22b7d196156be5fca0f2bd3badff0b9878e6..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/typescript/typescript.contribution.d.ts b/esm/vs/basic-languages/typescript/typescript.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/typescript/typescript.contribution.js b/esm/vs/basic-languages/typescript/typescript.contribution.js
deleted file mode 100644
index a0f2a73336417a9a495a3ecbb07790f40c6823c2..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/typescript/typescript.js b/esm/vs/basic-languages/typescript/typescript.js
deleted file mode 100644
index 8ddda1f346a8e345db7e51fa4aac7e7d1220617f..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/vb/vb.contribution.d.ts b/esm/vs/basic-languages/vb/vb.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/vb/vb.contribution.js b/esm/vs/basic-languages/vb/vb.contribution.js
deleted file mode 100644
index 4ea82e1b85c8834e10b3958ad0b8777b76e29652..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/vb/vb.js b/esm/vs/basic-languages/vb/vb.js
deleted file mode 100644
index 605d08985126d0ddd0bf3066a6020f4297da4d5b..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/wgsl/wgsl.contribution.d.ts b/esm/vs/basic-languages/wgsl/wgsl.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/wgsl/wgsl.contribution.js b/esm/vs/basic-languages/wgsl/wgsl.contribution.js
deleted file mode 100644
index 50cc19e4876fa8f10fbe28980e676b05252c70b6..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/wgsl/wgsl.js b/esm/vs/basic-languages/wgsl/wgsl.js
deleted file mode 100644
index 28811ba8736ec786a0e1fa48c59bb543cda20a50..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/xml/xml.contribution.d.ts b/esm/vs/basic-languages/xml/xml.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/xml/xml.contribution.js b/esm/vs/basic-languages/xml/xml.contribution.js
deleted file mode 100644
index 2eb3d0f30500d66a0de0faff208ec87ffa92857a..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/xml/xml.js b/esm/vs/basic-languages/xml/xml.js
deleted file mode 100644
index 6f8a391e4b36f49a4b5fd315eed16d1631cb2b4e..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/yaml/yaml.contribution.d.ts b/esm/vs/basic-languages/yaml/yaml.contribution.d.ts
deleted file mode 100644
index 336ce12bb9106afdf843063ee67c0c1970f70d37..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/yaml/yaml.contribution.js b/esm/vs/basic-languages/yaml/yaml.contribution.js
deleted file mode 100644
index 2f96b255bb38b95176b33a025b976fa3b742e470..0000000000000000000000000000000000000000
diff --git a/esm/vs/basic-languages/yaml/yaml.js b/esm/vs/basic-languages/yaml/yaml.js
deleted file mode 100644
index 4220fb0b5b38a697f870cd143b065d9468c85695..0000000000000000000000000000000000000000
